### DISCUSSION TITLE
Usability Fixes on Milestone view

### DISCUSSION THREAD

**Discussion by jmourinho** [Leantime/leantime#1367] [Ideas]
Hi,

some suggestions for improvement in the milestone view:

- Allow milestone bars to move up or down in the chart (drag and drop) in the gantt. As they are you do not have control on which milestones appear above or below others as they are organized by their starting date.
- When i want to scroll right to see the rest of the gantt the user always have to scroll down first to find the horizontal scroll bar of the gantt. Suggestion: define the max-height of the gantt to be inside the viewport, like setting its CSS to something like: _height: 80vh;_ or height: _calc(100vh - 100px);_
- Provide an Year view... for long projects it is better than doing the expensive task of scrolling aside
- The show tasks checkbox should use an ajax call to reload just the gantt control instead of reloading the whole page
- Allow Gantt export to PDF
- Allow milestone edit by double clicking it

**Comment by marcelfolaron** [comment]
Thank you for the feedback. I'll go ahead and create an issue for this to keep track of it.
